querybook/webapp/ui/FlowGraph/FlowGraph.tsx

```diff
@@ -178,6 +179,7 @@ const InteractiveFlowGraph: React.FunctionComponent<IGraphProps> = ({
                 edgeTypes={edgeTypes}
                 onInit={setGraphInstance}
                 fitView
+                maxZoom={1.2}
```

Comment:
why set zoom initially?

Reply:
with the `fitView` property set above, it will automatically zooming to fit view when the graph is loaded. `maxZoom` is just to make sure it will not zoom in too much.